### PR TITLE
Move datagov-terraform policy under Terraform

### PR DESCRIPTION
**Pull request by adborden**
This moves the CI user's IAM policy under version control.

- [x] `terraform import` the existing IAM policy before merging

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,28 @@ usage instructions in their respective README's. This section contains the usage
 instructions common to _all_ environments.
 
 
+### Setting up the CI deployer account
+
+Manually create the IAM CI deploy user (`datagov-ci`). The first execution of
+`iam` should be done manually with admin permissions. Once provisioned, the
```
Comment (mogul) [inline README.md:24]: Is there a specific command we can give here that will make this directive more accessible to Terraform newbs?

File: README.md [hunk 2]
```diff
@@ -52,16 +74,9 @@ You'll be prompted for your MFA code. Enter it without any spaces when prompted.
...
-[`iam`](iam/README.md) | Global "environment" that applies IAM settings to to the sandbox account. | v0.12 | N | N/A
+### Working with datagov-infrastructure-modules
 
-## Live Testing
-When doing live changes to the [modules](https://github.com/GSA/datagov-infrastructure-modules), you may have to change the source directory on terragrunt to be a local path. See [inventory 2.8](/ci/inventory-2-8/tarraform.tfvars#4) for an example.
\ No newline at end of file
+When making changes to the
+[datagov-infrastructure-modules](https://github.com/GSA/datagov-infrastructure-modules),
+you may have to change the source directory on terragrunt to be a local path.
+See [inventory 2.8](/ci/inventory-2-8/tarraform.tfvars#4) for an example.
```
Comment (mogul) [inline README.md:82]: This should be `/ci/inventory-2-8/terraform.tfvars#4` but this was already in the previous version so I won't let it block the review.

**Comment by mogul** [comment]
@adborden the [IAM plan diff](https://app.circleci.com/pipelines/github/GSA/datagov-infrastructure-live/150/workflows/18ef9fda-ed23-49c4-9cb4-f82a27db76db/jobs/698/parallel-runs/0/steps/0-102) is bigger than I expected, and I'm not sure what to make of that... Should we just expect this since there's been nothing to keep the existing IAM from drifting? 

Maybe more importantly, there's a NAT gateway change in [the CKAN cloud dev plan diff](https://app.circleci.com/pipelines/github/GSA/datagov-infrastructure-live/150/workflows/18ef9fda-ed23-49c4-9cb4-f82a27db76db/jobs/697/parallel-runs/0/steps/0-102) that makes me nervous. Can you take a look and make sure we're not stomping on some other change happening in parallel here?

File: README.md (hunk 1 above)
Comment (adborden) [reply README.md:24]: In theory, it's `terraform init && terraform apply`, but in practice if you were starting from scratch, you'd also need to create the s3 bucket which I don't want to get into.

Comment (adborden) [reply README.md]: Updated this section.

File: README.md (hunk 2 above)
Comment (adborden) [reply README.md]: Fixed the URL.

**Comment by adborden** [comment]
Thanks for calling out the plans. For IAM, I did some refactoring of the policy to hopefully make it more legible. I tried to organize permissions based on IAM administration, application administration, and boilerplate terraform state permissions. I also added a few ACM permissions to address [#53's failing build](https://circleci.com/gh/GSA/datagov-infrastructure-live/669).

For ckan-cloud-dev, I must have accidentally deleted the gateway recently. As part of https://github.com/GSA/datagov-infrastructure-live/pull/53 I ran into some limits in the AWS account and tried to delete old resources that are not managed by terraform. The ckan-cloud-dev NAT gateway was probably deleted by accident. Hopefully terrafrom will be able to re-create it without hitting the limit again but I will :baby: it and :sheep: it through.

**Pull request merged**